reference/5.1/Microsoft.PowerShell.Utility/Write-Host.md

```diff
@@ -77,8 +79,9 @@ Write-Host (2,4,6,8,10,12) -Separator ", -> " -ForegroundColor DarkGreen -Backgr
 2, -> 4, -> 6, -> 8, -> 10, -> 12
 ```
 
-This command displays the even numbers from two through twelve.
-It uses the `ForegroundColor` parameter to output 'dark green' text and the `BackgroundColor` parameter to display a 'white' background.
+This command displays the even numbers from two through twelve. It uses the `ForegroundColor`
+parameter to output 'dark green' text and the `BackgroundColor` parameter to display a 'white'
```

Comment:
I wouldn't put "dark green"  and "white" in quotes here, since the values aren't really quoted verbatim.